pandas/tests/reshape/merge/test_merge_asof.py

```diff
@@ -1339,3 +1339,24 @@ def test_merge_index_column_tz(self):
             index=pd.Index([0, 1, 2, 3, 4]),
         )
         tm.assert_frame_equal(result, expected)
+
+    def test_left_index_right_index_tolerance(self):
```

Comment:
add the issue number as a comment here
```suggestion
    def test_left_index_right_index_tolerance(self):
        # https://github.com/pandas-dev/pandas/issues/35558
```

Reply:
Thank you @simonjayhawkins